airflow/example_dags/example_setup_teardown_taskflow.py

```diff
@@ -47,12 +45,10 @@ def normal():
     @task_group
     def section_1():
         # You can also have setup and teardown tasks at the task group level
-        @setup
         @task
         def my_setup():
             print("I set up")
 
-        @teardown
```

Comment:
Should we create another example dag to demonstrate this usage instead of removing this one?

Reply:
yeah you are right